appengine/common/common.css

```diff
@@ -16,6 +16,9 @@ a:hover {
 .farSide {
   text-align: right;
 }
+#header_cta > * {
```

Comment:
Remove spaces around the > (to be consistent with similar rules in this file).

Reply:
Done.